src/routes/transactions/entities/readable-descriptions/erc20-descriptions.json

```diff
@@ -0,0 +1,5 @@
+{
+  "function transfer(address, uint256)": "Send {{tokenValue 1}} to {{address 0}}",
+  "function approve(address, uint256)": "Approve {{tokenValue 1}} to {{address 0}}",
+  "function withdraw(uint256)": "Withdraw {{tokenValue 0}}"
+}
```

Comment:
I would suggest to keep all of these templates in one file.

Reply:
Done, I've moved all of them into a single json.